templates/default/sv-mss-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Should these last two templates be file resources instead? There are no variables, nothing for ERB to do.